src/button/button.less

```diff
@@ -24,25 +24,29 @@
 
 .btn-primary {
     .btn();
-    background-color: @primary-color-dark;
+    background: @primary-color;
     color: @white;
 
     &:hover {
-        background-color: @primary-color-darker;
-        transition: background-color 300ms ease-out;
+        background-color: @primary-color-variant;
+        border: 1px solid @primary-color;
     }
 
     &[disabled] {
         background-color: @info-lighter;
         color: @gray-light;
         border: 1px solid @info-lighter;
-        transition: background-color 300ms ease-out;
     }
 }
 
 .btn-secondary {
     .btn();
     background-color: @white;
-    border-color: @info-dark;
-    color: @info-dark;
+    border-color: @primary-color;
+    color: @primary-color;
+
+    &:hover {
```

Comment:
@bedakb Looks like the `.btn-secondary` does not have styles for `&[disabled]` state, meaning on hover will change border/color even if it is disabled

Reply:
Fixed